daemon/logic/utils.go

```diff
@@ -96,12 +96,17 @@ func createCredentialGraph(ctx context.Context, credBody *PlaintextCredential,
 		return nil, err
 	}
 
-	team, err := findMembersTeam(teams)
+	membersTeam, machinesTeam, err := findSystemTeams(teams)
 	if err != nil {
 		return nil, err
 	}
 
-	memberships, err := client.Memberships.List(ctx, credBody.OrgID, team.ID, nil)
+	userMembers, err := client.Memberships.List(ctx, credBody.OrgID, membersTeam.ID, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	machineMembers, err := client.Memberships.List(ctx, credBody.OrgID, machinesTeam.ID, nil)
```

Comment:
Right now, destroyed machines are not removed from their teams, I can fix this quickly, but it'll require changes to the way `machines list` and `machines view` work (as they're expecting memberships to come back). 

Thoughts?